src/content/docs/en/guides/integrations-guide/markdoc.mdx

```diff
@@ -436,38 +450,8 @@ The following steps will create a custom Markdoc image tag to display a `<figure
     {% image src="./astro-logo.png" alt="Astro Logo" width="100" height="100" caption="a caption!" %}
     ```
 
-### Use client-side UI components
-
-Tags and nodes are restricted to `.astro` files. To embed client-side UI components in Markdoc, [use a wrapper `.astro` component that renders a framework component](/en/guides/framework-components/#nesting-framework-components) with your desired `client:` directive.
-
-This example wraps a React `Aside.tsx` component with a `ClientAside.astro` component:
-
-```astro title="src/components/ClientAside.astro"
----
-import Aside from './Aside';
----
-
-<Aside {...Astro.props} client:load />
-```
-
-This Astro component can now be passed to the `render` prop for any [tag][markdoc-tags] or [node][markdoc-nodes] in your config:
 
-```js title="markdoc.config.mjs"
-import { defineMarkdocConfig, component } from '@astrojs/markdoc/config';
-
-export default defineMarkdocConfig({
-  tags: {
-    aside: {
-      render: component('./src/components/ClientAside.astro'),
-      attributes: {
-        type: { type: String },
-      },
-    },
-  },
-});
-```
-
-### Markdoc config
+## Advanced Markdoc configuration
```

Comment:
Note for translators: this heading has changed! Not just moved! 

It was previously (lol) "### Markdoc config" (line 470 in English) nested under "## Markdoc config"

We heard you liked Markdoc config, so we put some Markdog config in your Markdoc config...